code/modules/map_fluff/cerestation.dm

```diff
@@ -0,0 +1,10 @@
+/datum/map/cerestation
+	name = "CereStation"
+	full_name = "NSS Cere"
```

Comment:
Same with this. Also needs a shorthand name thats not the map name as is the fashion. @Necaladun may have input here.